### ISSUE TITLE
mvix.sh fails to find packages on Tribblix 0m22

### ISSUE THREAD

**Issue by lonnietc** [ptribble/mvi#6]
Hi Peter,

Hope that you are doing well today.

I have just installed your Tribblix 0m22 (kitchen_sink) in a VirtualBox VM so that I can get a feel for things as well as to try and work with the mvi scripts after reading "Minimal Viable illumos" (https://ptribble.blogspot.com/2015/10/minimal-viable-illumos.html)

After installing Tribblix, I was able to use git to grab the mvi code from your github site and wanted to give it a try,

![image](https://user-images.githubusercontent.com/15264930/88821814-19b55a00-d191-11ea-9541-403703040922.png)

when I try:

./mvix.sh node vboxnet

I get an error about not being able to find some packages.

Also, on a side note, is it possible to install "mc" Midnight Commander on the Tribblix 0m22 as I use that utility a lot but did not see it in the "zap list-overlays" that I could choose?
Thanks

**Comment by ptribble** [comment]
Thanks for the feedback - mvi hasn't really been tried by many people other than myself, so it will need a few tweaks (and better documentation)

You probably want to set

THOME=${HOME}/test

as it isn't very good at finding things (and has a lot of assumptions about my own personal development machine).

You'll need to run things as root (that's partially a packaging constraint, but also to get the ownership of files in the resulting image correct).

And I think that if you don't have lots of other stuff checked out locally then you'll need the -s flag

./mvix.sh -s node vboxnet

**Comment by lonnietc** [comment]
Hi Again  Peter,

I just opened an xterm, used "su - root" to get into the root home directory, grabbed mvi from github, set the THOME at the shell, and then tried the build with the -s flag in place.

I also have mounted the Tribblix 0m22 cd in the Virtualbox drive, but still get the same error:

ERROR: unable to find mvi area /packages/localsrc/Tribblix/mvi

I do not think that the packages are setup since I also tried to follow a bit of 

https://jamesdeagle.blogspot.com/2017/07/tribblix-m20-getting-started-and.html

to get started and wanted to add some packages (like mc - Midnight Commander for easy console navigatio)  with "pkgin" but that also does not seem to be there.

**Comment by lonnietc** [comment]
Hi Peter,

Here is a quick update.

I was able to install the Joyent Package repo and then added some utilities to work with on mvi which I was able to compile a 30MB iso using:

 ./mvix.sh -s node vboxnet

Now I am interested in other basic ISO's that can be built with a single application.  

Just wondering, but what about with zlogin and or other zap packages?   Are packages from Pkgin also viable to try as well?

Of course, I would really like to see about adding bhyve and a simple login as a next test.
Cheers and truly thanks for the help.
